src/api/SearchApi.js

```diff
@@ -65,6 +66,48 @@ const SEARCH_TERM :string = 'searchTerm';
 const START :string = 'start';
 
 /**
+ * `GET /search/home/{start}/{maxHits}`
+ *
+ * Executes a search over all existing entity sets to populate the home page
+ *
+ * @static
+ * @memberof loom-data.SearchApi
+ * @param {number} start
+ * @param {number} maxHits
+ * @returns {Promise}
+ *
+ * @example
+ * SearchApi.loadHomePageEntitySets(0, 10);
+ */
+
+export function loadHomePageEntitySets(start :number, maxHits :number) :Promise<> {
```

Comment:
I wasn't sure whether to put it in EdmApi or SearchApi, but it's technically a search, and the previous endpoint (like loadPopularEntitySets or something like that) was also in SearchApi, so I went with that one